diskplorer.py

```diff
@@ -215,7 +247,20 @@ def global_section():
                     blocksize={args.write_buffer_size}
                     iodepth={args.write_concurrency}
                     rate={write_bw}
+                    {write_offset_str}
                     '''))
+
+                if args.run_sequential_trim:
+                    out(textwrap.dedent(f'''\
+                        [trim_data(r_idx={read_iops_step},w_idx={write_bw_step},write_bw={write_bw},r_iops={read_iops})]
+                        '''))
+                    out(textwrap.dedent(f'''\
+                        readwrite=trim
+                        blocksize={align_up(int(write_bw*args.trim_offset_time_seconds/2), 4096)}
```

Comment:
This is not a reasonable discard size, in ScyllaDB with ICS we delete 1GB files, even with 8-member RAID 0 it translates to 128MB discards.

Discards are rare+large. We could have a minimum discard size of (say) 32MB. If the write rate for a particular case is too low, the runtime should be extended so at least N (4?) events happen.

Reply:
Sure - an additional parameter that describes the minimum block size for trim and defaults to 32MiB has been added. Moreover, the code checks the number of trim operations that would be issued and extends the time if N<4 to ensure that at least N=4 operations are performed.